<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="dropLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Monospace&quot;;</string>
    </property>
    <property name="text">
     <string>Drop Shadow</string>
    </property>
   </widget>
   <widget class="QLabel" name="blurLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Monospace&quot;;</string>
    </property>
    <property name="text">
     <string>Blur Effect</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/butterfly.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/tux.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
